device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/qos.json.j2

```diff
@@ -1 +1,72 @@
+{%- macro clear_q_for_pcbb() %}
+    "DSCP_TO_TC_MAP": {
```

Comment:
We also need to define TC_TO_PRIORITY_GROUP_MAP for all these skus where 2->2 and 6->6

Reply:
Updated. I define a new `TC_TO_PRIORITY_GROUP_MAP` for all updated SKUs.